gluten-core/src/main/scala/io/glutenproject/expression/DecimalExpressionsTransformer.scala

```diff
@@ -0,0 +1,43 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.glutenproject.expression
+
+import com.google.common.collect.Lists
+import io.glutenproject.expression.ConverterUtils.FunctionConfig
+import io.glutenproject.substrait.expression.{ExpressionBuilder, ExpressionNode}
+
+import org.apache.spark.sql.catalyst.expressions.Expression
+
+class UnscaledValueTransformer(
```

Comment:
Please move this transformer to `UnaryExpressionTransformer.scala`

Reply:
it can reuse UnaryExpressionTransformer and don't need to create a new one, right ?

Reply:
Yes
```
case class UnscaledValue(child: Expression) extends UnaryExpression
```

Reply:
@liujiayi771 Can you help to resolve this comment? Thanks.

Reply:
I have removed these codes.